src/ui/RadioButton.js

```diff
@@ -36,25 +37,30 @@ const RadioInputInputStyle = styled.input.attrs({ type: 'radio' })`
   :focus {
     outline: none;
   }
+
+  :disabled {
+    background-color: #dddddd;
+  }
 `
 
-const RadioInputTextStyle = styled.label`
-  margin-left: 0.5rem;
```

Comment:
Should we be moving from `rem` to `px`? RadioButton hasn't been used before in Vavato I think :thinking:

Reply:
Actually, It's not a good practice to use `rem` in margin/padding, if the reference font size change it can break the element.